app/frontend/src/services/commonFormService.js

```diff
@@ -48,19 +279,7 @@ export default {
    * @returns {Promise} An axios response
    */
   requestTeamAccess(form) {
+    if (!isValidForm(form)) return Promise.reject('Invalid form specified');
```

Comment:
this is called a lot.  can't we just write a wrapper around getting appAxios?
getAxios(form) that will do the check and reject?  something like that, have it done and handled in one place.

Reply:
could even do it with like a service factory, pass the form name to the service factory and reject it immediately.  few ways to slice it and reduce the repetition.

Reply:
Yeah having this same line everywhere must be something we can get around...

Reply:
I've updated this by wrapping appAxios function and throwing if invalid.